internal/agents/qwen/usage.go

```
@@ -0,0 +1,479 @@
+	}
+	if len(unique) == 0 {
+		return nil
+	}
+	return errors.New(strings.Join(unique, "\n"))
+}
+
+func shellQuoteArgument(value string) string {
+	return "'" + strings.ReplaceAll(value, "'", "'\"'\"'") + "'"
+}
+
+func readConsoleConfig(accountID string) (consoleConfig, error) {
```

golangci-lint (2.12.2)

[error] func readConsoleConfig is unused

(unused)